python/baseline/remote.py

```diff
@@ -40,17 +42,24 @@ def predict(self, examples):
         :param examples: The input examples
         :return: The outcomes
         """
-        import requests
+        import httplib, urllib
```

Comment:
The `urllib` import isn't needed and the import can be moved to the top of the file

Reply:
this looks like it was moved to the top, but it never should be imported AFAICT